src/test-utils/consoleErrorMock.spec.js

```diff
@@ -0,0 +1,34 @@
+/* eslint-disable flowtype/require-valid-file-annotation, no-console */
+
+import { assert } from 'chai';
+import consoleErrorMock from './consoleErrorMock';
+
+describe('consoleErrorMock()', () => {
+  it('should throw error when calling callCount() before spy()', () => {
+    let errorCalledFlag = false;
+
+    try {
+      consoleErrorMock.callCount();
+    } catch (error) {
+      errorCalledFlag = true;
+    }
+
+    assert.strictEqual(errorCalledFlag, true);
+  });
+
+  describe('spy()', () => {
+    it('should place a spy in console.error', () => {
+      consoleErrorMock.spy();
+      assert.ok(console.error.hasOwnProperty('isSinonProxy'));
```

Comment:
I'm wondering `hasOwnProperty` returns a boolean right? Should we use `strictEqual`?